t/comp/proto.t

```diff
@@ -892,3 +892,20 @@ $_ = sub ($$$$$$$) {};
 @_ = (1, 2, 3, prototype(), 4, 5, 6);
 print "not " unless "@_" eq '1 2 3 $$$$$$$ 4 5 6';
 print "ok ", $i++, " - [perl #123514] (got @_)\n";
+
+{
+    $i++;
+    my $failure = q<>;
+
+    if (eval 'sub scalarref (\$) { }; scalarref( sub {} ); 1') {
+        print "not ok $i: Unexpected scalarref success!\n";
+    }
+    else {
+        if ($@ !~ m/anonymous subroutine/ || $@ !~ m/scalarref/) {
+            $failure = $@;
+            print "Unexpected error (or none): $failure";
```

Comment:
this can confuse the harness. Stick a # at the start, if $failure can be multiline stick a # in front of each.

Reply:
Fixed.